packages/angular/cli/commands/update.json

```diff
@@ -62,6 +62,12 @@
           "description": "Display additional details about internal operations during execution.",
           "type": "boolean",
           "default": false
+        },
+        "skipCommits": {
+          "description": "Do not create source control commits for updates and migrations.",
+          "type": "boolean",
+          "default": false,
+          "aliases": ["C"]
```

Comment:
Nit: won't it be better to use `sc` as an aliases? So that it is not confused with `-c` `collection`

Reply:
Aliases should only be a single letter.  `ng update -vC` is a valid option set for verbose and no commits.  Also, `collection` is not an option for update.